src/controllers/requestController.js

```diff
@@ -1,21 +1,43 @@
-/* eslint-disable no-await-in-loop */
 /* eslint-disable guard-for-in */
 /* eslint-disable no-restricted-syntax */
-/* eslint-disable max-len */
 import db from '../database/models';
 import sendResult from '../utils/sendResult';
 
 const Request = {
   async getRequest(req, res) {
     try {
-      const requests = await db.Requests.findAll({ where: { UserId: req.userId }, raw: true });
+      const requests = await db.Requests.findAll({ where: { UserId: req.user.userId }, raw: true });
       for (const index in requests) {
         requests[index].Trips = await db.Trips.findAll({ where: { RequestId: requests[index].id }, raw: true, attributes: { exclude: ['RequestId'] } });
       }
```

Comment:
As the request and trip table are linked, why don't you use ``include`` statement in the first query. It will return requests with trips embedded.